tpot/base.py

```diff
@@ -40,16 +40,16 @@
 from shutil import rmtree
 
 import numpy as np
-from pandas import DataFrame
+from pandas import DataFrame, Series
 from scipy import sparse
 import deap
 from deap import base, creator, tools, gp
 from copy import copy, deepcopy
 
 from sklearn.base import BaseEstimator
 from sklearn.utils import check_X_y, check_consistent_length, check_array
-from sklearn.pipeline import make_union
-from imblearn.pipeline import make_pipeline
+from sklearn.pipeline import make_union, make_pipeline
+from imblearn.pipeline import make_pipeline as make_imblearn_pipeline
```

Comment:
Please use `try...except...` for environment without `imblearn`

Reply:
now added